### PR TITLE
Rebecca docs

### PR DESCRIPTION
**Pull request by criticalglitch**
Doxygen doc for my changes to the model, window, player and block classes. 

Advances Issue #70 and #71.

### CODE REVIEW

**Comment by bakursait2** [comment]
The documentations looks good. I did some changes in my side as well. So I guess I will make a branch out of yours to keep everything updated.

**Review by bakursait2 — APPROVED** [review]
The documentations looks good. I did some changes on my end as well. So I guess I will make a branch based on your version to keep everything updated.

**Review by Lurk390 — APPROVED** [review]
Documentation looks really good. I will match your style when I do it myself.

File: tempus_fugit_minecraft/model.py
Comment (w1158mxs) [inline tempus_fugit_minecraft/model.py:1]: Give me Load diff!

**Review by w1158mxs — APPROVED** [review]

**Review by xieWright — CHANGES_REQUESTED** [review]
There are some merge conflicts, fix before we merge this.

**Review by xieWright — APPROVED** [review]

**Pull request merged**